src/sentry_plugins/clubhouse/plugin.py

```diff
@@ -0,0 +1,95 @@
+
+import six
+
+from sentry.exceptions import PluginError
+from sentry.plugins.bases.issue2 import IssuePlugin2, IssueGroupActionEndpoint
+
+from sentry_plugins.base import CorePluginMixin
+from sentry_plugins.utils import get_secret_field_config
+
+from .client import ClubhouseClient
+
+
+class ClubhousePlugin(CorePluginMixin, IssuePlugin2):
+    # TODO: There currently isn't a good "list all story titles" endpoint for Clubhouse.
+    # When it does, implement a `link` action, and remove the below line.
+    allowed_actions = ('create', 'unlink')
+
+    description = 'Create Clubhouse Stories from a project.'
+    slug = 'clubhouse'
+    title = 'Clubhouse'
+    conf_title = title
+    conf_key = 'clubhouse'
+
+    def get_group_urls(self):
+        return super(ClubhousePlugin, self).get_group_urls() + [
+            (
+                r'^autocomplete', IssueGroupActionEndpoint.as_view(
+                    view_method_name='view_autocomplete',
+                    plugin=self,
+                )
+            ),
+        ]
+
+    def get_configure_plugin_fields(self, request, project, **kwargs):
+        token = self.get_option('token', project)
+        helptext = (
+            'Enter your API Token (found on '
+            'your account Settings, under API Tokens).'
+        )
+        secret_field = get_secret_field_config(token, helptext, include_prefix=True)
+        secret_field.update(
+            {
+                'name': 'token',
+                'label': 'API Token',
+                'placeholder': 'e.g. 12345678-1234-1234-1234-1234567890AB'
+            }
+        )
+        return [
+            secret_field, {
+                'name': 'project',
+                'label': 'Project ID',
+                'default': self.get_option('project', project),
+                'type': 'text',
+                'placeholder': 'e.g. 639281',
+                'help': 'Enter your project\'s numerical ID.'
+            }
+        ]
+
+    def is_configured(self, request, project, **kwargs):
+        return all(self.get_option(k, project) for k in ('token', 'project'))
+
+    def create_issue(self, request, group, form_data, **kwargs):
+        client = ClubhouseClient()
+
+        try:
+            response = client.create_story(
+                project=self.get_option('project', group.project),
+                data=form_data,
+                token=self.get_option('token', group.project)
+            )
+        except Exception as e:
+            self.raise_error(e)
+
+        return response['id']
+
+    def get_issue_label(self, group, issue_id, **kwargs):
+        return 'Clubhouse Story #%s' % issue_id
+
+    def get_issue_url(self, group, issue_id, **kwargs):
+        # TODO: Uncertain of this is a good method, as it incurs an API call on page load.
+        # `issue_url` can't be generated purely from a knonwn string + `issue_id`, as it
+        # requires the Clubhouse Organization in the URL, which isn't currently a known API call.
```

Comment:
And this too.

Reply:
I never got around to finishing this:

https://github.com/getsentry/sentry/pull/6250

Reply:
@eliperkins Thanks - I considered that, and since I _think_ the org name can be altered, that would make this connection more brittle.

@dcramer Do you think you might get back to it soon, or would this be an acceptable first implementation for this integration?

My thoughts are that since the API call will only occur when loading a Sentry issue that is _already linked_ to a Clubhouse Story, the volume will be minimal, and could be shipped sooner.
I could add a comment with a link to that issue, to revisit when the core sentry change is implemented.

Reply:
Let me take a look and see. Otherwise you could store the value as something JSON encoded, and then decode it when it comes out (allowing you to store multiple values).